Library/Formula/kubernetes-cli.rb

```diff
@@ -4,13 +4,8 @@ class KubernetesCli < Formula
   head "https://github.com/kubernetes/kubernetes.git"
 
   stable do
-    url "https://github.com/kubernetes/kubernetes/archive/v1.0.6.tar.gz"
-    sha256 "cb6d03ddc920ef5eca13d12688419dff0b21daf3bf45ad731bfcae57f15ce8ea"
-
-    patch do
-      url "https://github.com/kubernetes/kubernetes/commit/defe0f82bf99ce2ec5179d92475c00ce9beaf318.patch"
-      sha256 "80e0267461f778b51a21f5876152de5e74730b7ffb77c7604fe1b63b7983d3ba"
-    end
+    url "https://github.com/kubernetes/kubernetes/archive/v1.1.1-beta.1.tar.gz"
```

Comment:
If this is a beta it should go inside a `devel do` block rather than punching everyone from the stable branch onto an unstable release.